lib/timestamps.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function timestamps (data) {
+	const format = (year, month, day) => {
+		const monthName = new Date(year, month)
+			  .toLocaleDateString('en-GB', { month: 'short' })
+			  .toUpperCase();
+		return `${monthName} ${day}, ${year}`;
```

Comment:
'en-US' formats the date in the order time stamps appear across the board ("month dd, yyyy")